preview/app/controllers/FaciaDraftController.scala

```diff
@@ -1,25 +1,28 @@
-package controllers.preview
+package controllers
 
+import com.gu.contentapi.client.model.v1.ItemResponse
 import contentapi.{ContentApiClient, SectionsLookUp}
-import controllers.{FaciaController, IndexController}
-import controllers.front.FrontJsonFapi
+import controllers.front.FrontJsonFapiDraft
 import model.ApplicationContext
-import play.api.libs.ws.WSClient
+import play.api.mvc.{RequestHeader, Result}
 import services.ConfigAgent
 
-class FrontJsonFapiDraft(val wsClient: WSClient) extends FrontJsonFapi {
-  val bucketLocation: String = s"$stage/frontsapi/pressed/draft"
-}
+import scala.concurrent.Future
 
-class FaciaDraftController(val frontJsonFapi: FrontJsonFapi, contentApiClient: ContentApiClient, sectionsLookUp: SectionsLookUp)(implicit val context: ApplicationContext) extends FaciaController {
+class FaciaDraftController(val frontJsonFapi: FrontJsonFapiDraft, contentApiClient: ContentApiClient, sectionsLookUp: SectionsLookUp)(implicit val context: ApplicationContext) extends FaciaController with RendersItemResponse {
 
   private val indexController = new IndexController(contentApiClient, sectionsLookUp)
 
-  override def renderFront(path: String) = {
+  override def renderItem(path: String)(implicit request: RequestHeader): Future[Result] = {
     log.info(s"Serving Path: $path")
+
     if (!ConfigAgent.getPathIds.contains(path))
-      indexController.render(path)
+      indexController.renderItem(path)
     else
-      renderFrontPress(path)
+      renderFrontPressResult(path)
   }
+
+  override def canRender(path: String): Boolean = ConfigAgent.getPathIds.contains(path)
+
+  override def canRender(item: ItemResponse): Boolean = indexController.canRender(item)
```

Comment:
Preview `FaciaDraftController` class was actually not used anywhere. This is just replacing it by the `Standalone` one.